apps/folderlaunch/types.d.ts

```diff
@@ -0,0 +1,48 @@
+type AppInfoFile = {                // Contents of a .info file
```

Comment:
If you like, we could reuse `info.d.ts`'s [`AppInfo` type](https://github.com/espruino/BangleApps/blob/master/typescript/types/info.d.ts), I suppose we'd want to add `id`, `data`, `files`, `tags` and `version` to it (all as optional)

Reply:
I like that idea. Can we just do that, or is there a chance of breaking apps that rely on the AppInfo type? I don't *think* so because we're just adding stuff to it, but I just want to be sure.

Reply:
Seems like popconlaunch is the only other app that uses this?

Reply:
Great, thanks! Yeah that sounds right to me, and adding shouldn't be a problem in this case either

Reply:
OK, I have added to the type.

Reply:
Great, thanks! I've just checked and going off [`createAppJSON`](https://github.com/espruino/EspruinoAppLoaderCore/blob/770b1b71f16399eb3eb86a4c3198e1fbed979110/js/appinfo.js#L243-L280), I think we can make `tags`, `type` and `version` optional too, wdyt?

Reply:
I'm surprised that the app loader allows apps to not have a type and version, but I agree that if it's possible, the type should reflect it.

Reply:
Yeah the version especially, although perhaps it's for local development or custom apps